math/armstrong_number.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file
+ * \brief A C++ program to check whether a number is armstrong number or not.
+ * 
+ * \details
+ * Armstrong number or [Narcissistic number](https://en.wikipedia.org/wiki/Narcissistic_number) 
+ * is a number that is the sum of its own digits raised to the power of the number of digits.
+ * @author iamnambiar
+*/
+#include<cassert>
+#include<iostream>
+#include<cmath>
```

Comment:
```suggestion
#include <cassert>
#include <iostream>
#include <cmath>
```

Reply:
> Suggested change 
> #include<cassert>
> #include<iostream>
> #include<cmath>
> #include <cassert>
> #include <iostream>
> #include <cmath>

with the new GitHub-actions, such code-formatting would be automatically committed :)
but the author has to enable the GitHub actions for this repo.

Reply:
means should we include lgtm to forked repo @kvedala from setting in integration tab?

Reply:
> means should we include lgtm to forked repo @kvedala from setting in integration tab?

nope. the `clang-format` already exists on the main repo branch GitHub-actions